docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md

```diff
@@ -0,0 +1,134 @@
+
+- Time-boxing and focus/productive time
+- Consider the impact of work you’re doing
+- Nurture and focus on the community, when you can
+- Take breaks
+- Productive interruptions
+- Reserve time for play
+- Know when to pivot, and ignore all of the above
+
+Important caveat: treat this as a guide--it is by no means exhaustive, nor representative of how each individual works most effectively, but it does serve as a reasonably accurate model of a productive core maintainer.
+
+## What exactly is productive work, anyways?
+
+Before we get started, let's take a step back and ask: what exactly is "productive work" for a core maintainer?
+
+In general, a non-exhaustive list of productive work: is work that drives valuable outcomes for…
+
+- the community
+- Gatsby Core, or
+- other teams at Gatsby.
+
+An ideal scenario is work that intersects with all three, or that unblocks or leads to future work that can reach a wider audience.
+
+Some examples:
+
+- **Shipping roadmap features**. Many Roadmap features are usually high-effort but often - hugely impactful in helping large numbers of Gatsby users.
+- **Shipping meaningful, small bug-fixes**. Many small bug-fixes are low-effort and medium-impact. They "move the needle" and improve the day-to-day experience of using - Gatsby for a non-trivial amount of users.
+- **Reviewing PRs**. Giving timely, thorough, and fair reviews for pull requests from both the community and other Gatsby Core team members unblocks future work or changes required for merging, and delivers or eventually delivers meaningful fixes or improvements for - the community, at large.
+- **Helpful, targeted use of expertise**. Unblocking a community member or Gatsby team member, which may take just a few minutes of your time may unblock and drastically reduce the time spent on a particularly tough issue by an ally. If someone surfaces a question in - Slack or Discord—if you have a few moments, help them out!
+- **Pairing**. Leveraging your skill set to help another member of the team is a great way to - get a meaningful feature delivered more quickly. Hop on a Zoom and build something great!
+- **Writing**. Writing docs, writing a blog post, speaking at a conference or on a podcast, - cultivating community members to level-up, etc. [See “Why we Write” by Sam Bhagwat](https://www.gatsbyjs.org/blog/2019-04-19-gatsby-why-we-write/).
+- **Creating issues**. Create and/or edit issues, RFCs, etc. for community members to discuss or pick up. If we can let other people work on relatively simpler tasks, it frees us up to work on higher-value tasks, which tend to be more complex and require more focused - effort.
```

Comment:
> work on relatively simpler tasks
I don't love this phrase, but I also don't have suggestion on how to rephrase that. 

Maybe something more in the lines of "If we can get help from the community with shipping features, fixing bugs, and improving documentation, it frees us up to work on large-scale projects"?

Reply:
^ way better way of saying it. I'll address.